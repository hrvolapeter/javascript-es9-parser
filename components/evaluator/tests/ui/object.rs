var a = {a: 'a', b: 1};
var b = {a: a, b: 'b'};
console.log(a);
console.log(a.a);
console.log(b);
