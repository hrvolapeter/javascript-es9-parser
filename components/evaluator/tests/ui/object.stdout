{a: "a", b: 1}
"a"
{a: {a: "a", b: 1}, b: "b"}
